exe/cli/cmd/ffs_watch.go

```diff
@@ -0,0 +1,143 @@
+package cmd
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"os/signal"
+	"sort"
+	"strings"
+	"syscall"
+
+	"github.com/gosuri/uilive"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"github.com/textileio/powergate/api/client"
+	"github.com/textileio/powergate/ffs"
+)
+
+func init() {
+	ffsWatchCmd.Flags().StringP("token", "t", "", "FFS auth token")
+
+	ffsCmd.AddCommand(ffsWatchCmd)
+}
+
+var ffsWatchCmd = &cobra.Command{
+	Use:   "watch [jobid,...]",
+	Short: "Watch for job status updates",
+	Long:  `Watch for job status updates`,
+	PreRun: func(cmd *cobra.Command, args []string) {
+		err := viper.BindPFlags(cmd.Flags())
+		checkErr(err)
+	},
+	Run: func(cmd *cobra.Command, args []string) {
+		ctx := context.Background()
+
+		if len(args) != 1 {
+			Fatal(errors.New("you must provide a comma-separated list of job ids"))
+		}
+
+		idStrings := strings.Split(args[0], ",")
+		jobIds := make([]ffs.JobID, len(idStrings))
+		for i, s := range idStrings {
+			jobIds[i] = ffs.JobID(s)
+		}
+
+		state := make(map[string]*client.JobEvent, len(jobIds))
+		for _, jobID := range jobIds {
+			state[jobID.String()] = nil
+		}
+
+		writer := uilive.New()
+		writer.Start()
+
+		updateJobsOutput(writer, state)
+
+		ch, cancel, err := fcClient.Ffs.Watch(authCtx(ctx), jobIds...)
+		checkErr(err)
+		defer cancel()
+
+		c := make(chan os.Signal)
+		signal.Notify(c, os.Interrupt, syscall.SIGTERM)
+		go func() {
+			<-c
+			cancel()
+			os.Exit(0)
+		}()
```

Comment:
Trying to make sure we capture Ctrl+C here and cancel the listener back on the server.